### PR TITLE
use evm-tests without lock file

### PR DESCRIPTION
**Pull request by ermalkaleci**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -215,15 +215,15 @@ cargo-update:
 
 .PHONY: build-wasm-mandala
 build-wasm-mandala:
-	./scripts/build-only-wasm.sh --profile production -p mandala-runtime --features=on-chain-release-build --workspace --exclude runtime-integration-tests --exclude e2e-tests
+	./scripts/build-only-wasm.sh --profile production -p mandala-runtime --features=on-chain-release-build --package mandala-runtime
```
Comment (xlc) [inline Makefile:218]: why this change? there is already `-p mandala-runtime`

**Review by xlc — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (ermalkaleci) [reply Makefile:218]: Because all we want is the mandala-runtime, so it has to compile one package and not entire workspace

Comment (ermalkaleci) [reply Makefile:218]: Old command will build relaychain runtime as well

Comment (ermalkaleci) [reply Makefile:218]: Oh I think I missed it

Comment (ermalkaleci) [reply Makefile:218]: Will fix tomorrow

**Review by xlc — APPROVED** [review]
